koekalenteri-frontend/src/pages/SearchPage.tsx

```diff
@@ -12,44 +12,46 @@ import { FilterProps } from '../stores/PublicStore';
 export const SearchPage = observer(function SearchPage() {
   const { rootStore, publicStore } = useStores();
   const [sessionStarted, setSessionStarted] = useSessionStarted();
+
   const [searchParams, setSearchParams] = useSearchParams();
   const [filter, setFilter] = useState(deserializeFilter(searchParams));
 
   const organizers = toJS(rootStore.organizerStore.organizers);
-  const judges = toJS(rootStore.judgeStore.activeJudges);
-  const eventTypes = rootStore.eventTypeStore.activeEventTypes.map(et => et.eventType);
+  const judges = rootStore.judgeStore.activeJudges.map(j => { return {...j}});
```

Comment:
Fun fact: mobx:n observablella ja computed valuella näyttää olevan hienoisia eroja niiden jatkokäsittelyssä. Jos activeJudgesin koittaa konvertoida tavalliseksi JS dataksi toJS:llä, taulukon sisällä olevat objectit edelleen säilytivät observableina. Tällä kotkautuksella saadaan myös objectit plain JS:ksi.

Reply:
toi [toJSON](https://github.com/koekalenteri/koekalenteri/blob/44792a327be4522977c5c6331617375c8980fbee/koekalenteri-frontend/src/stores/classes/CJudge.ts#L42) pitäis mun käsittääkseni toimia `toJS` kanssa.